### ISSUE TITLE
Add tags for WCAG 2.1 SC

### ISSUE THREAD

**Issue by yatil** [w3c/wai-wcag-quickref#166]
@jake-abma via email:

> Hi Eric / all,
>
> When filtering on ‘zoom’ I suggest Reflow 1.4.10 should be added to the filter results.
>
> https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_customize&tags=zoom
>
> Cheers!
> Jake
> 
> Regards,
> Jake Abma

Actually, we don’t have any tags for the new WCAG 2.1 SCs. We should have tags for them. 

(FTR: Tags can be added in this file: https://github.com/w3c/wai-wcag-quickref/blob/gh-pages/_data/tags-sc.yml)

**Comment by shawna-slh** [comment]
This is on the list of things to do for WCAG 2.1.

@awkawk , @alastc , @sharronrush , @bakkenb ,
A sub-set of folks from EOWG did the tags last time, afaik. I don't think we have enough folks in EOWG who are familiar enough with the new 2.1 SCs to do this. I think AG folks need to do this, maybe folks from the TFs.

@yatil, Please provide clear direction on specifically what is needed. My guess is:
* For each of the new SCs, list relevant tags. Add them to https://github.com/w3c/wai-wcag-quickref/blob/gh-pages/_data/tags-sc.yml
* Here is the list of [handles?] for the new SCs: @@ [or maybe it's best if you add them to the yml file?]
* Here is the list of available tags in the four categories: @@
* (optional) If you'd like us to consider adding new tags, @@[send proposal where?] (we'd rather not, so please only do this if you feel it is worth the effort)

Sharron & Eric, I think you were on the team who did the tags for 2.0. Do you have any hard or soft guidance on assigning tags for SCs? Is there someone else who was on that team that we might want to ask?

Alastair & Andrew, Please confirm that AG will take this on.

Thanks all!

p.s. /me apparently can't assign awkawk & alastc

**Comment by awkawk** [comment]
Someone is going to need to explain what actually needs to be done:

What are "dev", "int", "vis", and "con"?

non-text-content:
  dev: captcha images text-alternatives video
  int: audio video buttons carousels captcha
  vis: images images-of-text animation progress-steps text-alternatives video
  con: audio captions content images live-stream moving-content text-alternatives video visual-cues buttons

**Comment by alastc** [comment]
From the [quickref](https://www.w3.org/WAI/WCAG21/quickref/), I'd guess they drive the left-nav:

- Developing
- Interaction Design
- Content Creation
- Visual Design 

But it would help to have a complete list of the possible values, and know whether the order matters?

I.e. stick them on the end, or put in SC/POUR order?

**Comment by yatil** [comment]
Urgh. Yes, @awkawk. I wanted to provide more context before pinging y’all.

`dev`, `int`, `vis` and `con` are the different tag categories:

* Developing
* Interaction Design
* Visual Design
* Content Creation

The tags are free form (hence tags) and there is no complete list (we can add new) although EOWG bundled and simplified similar tags. The tag order in the document does not matter, they are sorted alphabetically when they are yielded in the Quickref.

I can't give more context today as I’m supposed to be in a meeting right now.

**Comment by shawna-slh** [comment]
Task moved to https://github.com/w3c/wai-wcag-quickref/pull/181/files
